README.md

```diff
@@ -40,8 +40,16 @@ Build lobid-resources:
 
 Build the web application:
 
-- `cd web`
-- Then follow the script in `web/monit_restart.sh`.
+Check if `$JAVA_HOME` variable is set
+- `echo $JAVA_HOME`
+
+Set the variable to the home folder, not the path of your JAVA installation:
+e.g.: `export JAVA_HOME="/usr"`
+
+- `cd app/transformation/input/`
+- `sbt clean`
+- `sbt --java-home $JAVA_HOME stage`
+- `JAVA_OPTS="$JAVA_OPTS -XX:+ExitOnOutOfMemoryError" ./target/universal/stage/bin/lobid-organisations -Dhttp.port=9000 -no-version-check`
```

Comment:
- I think the addition to JAVA_OPTS is not required for regular local startup (like setting JAVA_HOME, it's something we do in our production setup)
- The binary is from lobid-organisations
- 9000 is the default port, no need to set it here